app/managers.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return actors_cursor.fetchall()
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name)"
+            " VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def update(
+            self, id_to_update: int, new_first_name: str, new_last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} SET first_name = ?, last_name = ?"
```

Comment:
If you have a faintest shadow of compassion, you should follow the SQL guidelines. 
I beg of you to have mercy on the souls of those reading your code and always start clauses from new lines while writing raw SQL. 
It may not be very important here, but it really is better to get used to the good practices before it's too late and someone — maybe even your future self — gets hurt. 

This applies `SET` clause here and also `WHERE` at $35.

Reply:
You're right) Fixed
I'm getting used to this)